mappings/net/minecraft/world/gen/feature/ConfiguredFeature.mapping

```diff
@@ -8,6 +8,8 @@ CLASS net/minecraft/unmapped/C_stwyylfs net/minecraft/world/gen/feature/Configur
 	METHOD <init> (Lnet/minecraft/unmapped/C_lowstvki;Lnet/minecraft/unmapped/C_jpnewgxf;)V
 		ARG 1 feature
 		ARG 2 config
+	METHOD equals (Ljava/lang/Object;)Z
```

Comment:
`m_ifdiewua` and `m_wrlrbrkd` should be renamed to `feature` and `config` respectively

Reply:
Why? This class isn't a record

Reply:
Ah, I thought it was one, my bad